src/utils/api/apiFetch.ts

```diff
@@ -4,7 +4,7 @@ import type { ApiResponse, QueryType } from "../interfaces";
 import { ApiExecution } from "./apiConstructor";
 import { fetchData } from "./apiFunctions";
 import { EventResponseType } from "./events/interfaces";
-import { LifeCenterType } from "./lifeCenter/interfaces";
+import { LifeCenterDetailsType, LifeCenterType } from "./lifeCenter/interfaces";
```

Comment:
please normalise running ALT + shft + O to format imported modules

let talk about editting your settings json to do this once you save in the office tomorrow